rs/ethereum/cketh/minter/tests/subaccount.rs

```diff
@@ -1,47 +1,18 @@
 use candid::Nat;
-use ic_base_types::PrincipalId;
 use ic_cketh_minter::memo::MintMemo;
 use ic_cketh_test_utils::ckerc20::{CkErc20Setup, DepositCkErc20WithSubaccountParams, ONE_USDC};
 use ic_cketh_test_utils::flow::{
     DepositCkEthParams, DepositCkEthWithSubaccountParams, DepositParams,
 };
 use ic_cketh_test_utils::{
-    CkEthSetup, DEFAULT_DEPOSIT_FROM_ADDRESS, DEFAULT_DEPOSIT_LOG_INDEX,
-    DEFAULT_DEPOSIT_TRANSACTION_HASH, DEFAULT_ERC20_DEPOSIT_LOG_INDEX,
-    DEFAULT_ERC20_DEPOSIT_TRANSACTION_HASH, DEFAULT_PRINCIPAL_ID, DEFAULT_USER_SUBACCOUNT,
-    EXPECTED_BALANCE,
+    DEFAULT_DEPOSIT_FROM_ADDRESS, DEFAULT_DEPOSIT_LOG_INDEX, DEFAULT_DEPOSIT_TRANSACTION_HASH,
+    DEFAULT_ERC20_DEPOSIT_LOG_INDEX, DEFAULT_ERC20_DEPOSIT_TRANSACTION_HASH,
+    DEFAULT_USER_SUBACCOUNT, EXPECTED_BALANCE,
 };
 use icrc_ledger_types::icrc1::account::Account;
 use icrc_ledger_types::icrc1::transfer::Memo;
 use icrc_ledger_types::icrc3::transactions::Mint;
 
-#[test]
-fn should_deposit_and_withdraw_cketh() {
```

Comment:
Since the test would be almost exactly like the one in `rs/ethereum/cketh/minter/tests/cketh.rs` I decided to move it there.